docs/src/basics/client-server.md

```diff
@@ -1,25 +1,48 @@
-# Client / Server
+# Architecture
 
-Zeebe uses a client / server architecture. There are two main components: *Broker* and *Client*.
+There are four main components in Zeebe's architecture: the client, the gateway, the broker, and the exporter.  
 
-![client-server](/basics/client-server.png)
+![zeebe-architecture](/basics/zeebe-architecture.png)
+
+## Client
+
+Clients are libraries that you embed in an application (e.g. a microservice that executes your business logic) to connect to a Zeebe cluster. Clients have two primary uses:
+* Carrying out business logic (starting workflow instances, publishing messages, working on tasks)
+* Handling operational issues (updating workflow instance payloads, resolving incidents)
+
+More about Zeebe clients:
+* Clients connect to the Zeebe gateway via [gRPC](https://grpc.io), which uses http/2-based transport.
```

Comment:
The gRPC docs are being merged at the moment, maybe link to the new page here as well?